### PR TITLE
Styled components

### PR DESCRIPTION
**Pull request by BelaM80**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import Page from "./components/pages/Page.js";
-import "./App.css";
+import GlobalStyle from "./GlobalStyle";
 import React from "react";
+import Input from "../src/components/Input/Input";
 
 function App() {
   return (
     <div className="App">
```
Comment (macfrei) [inline client/src/App.js:8]: You don't need the className anymore when using styled-components and you already deleted your `App.js`

File: client/src/components/Input/Input.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
```
Comment (macfrei) [inline client/src/components/Input/Input.js:1]: You could import it like this instead: `import styled from "styled-components/macro"` This way the generated class name will include the component name

**Review by macfrei — COMMENTED** [review]
Nicely done! Just added minor comments you could consider. If you have any question about that, please get back to me. 🚀

File: client/src/components/Input/Input.js (hunk 2 above)
Comment (BelaM80) [reply client/src/components/Input/Input.js:1]: 👍🏻 good suggestion! I will do that😃

**Review by lmachens — APPROVED** [review]
💯

File: package.json [hunk 3]
```diff
@@ -54,6 +55,7 @@
   "dependencies": {
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "postinstall": "^0.7.0"
+    "postinstall": "^0.7.0",
```
Comment (lmachens) [inline package.json:58]: Not related to this Pull Request, but  this module should not be required.

Comment (BelaM80) [reply package.json:58]: Ok, but I think, this might have everyone from your template..😄

File: client/src/App.js (hunk 1 above)
Comment (BelaM80) [reply client/src/App.js:8]: What do you mean by "already deleted your app.js"?

Comment (BelaM80) [reply client/src/App.js:8]: In this case, I do need the App class, because I call it in the Global Style. Or am I wrong with this?

Comment (macfrei) [reply client/src/App.js:7]: I meant `App.css` sorry for that. And I guess you can do that, however I am not quite sure whether that is best practice or not.

**Pull request merged**